lib/measures/SetCOPforSingleSpeedDXCoolingUnits/measure.rb

```diff
@@ -268,8 +268,8 @@ def get_total_costs_for_objects(objects)
         if !hVACComponent.empty?
           hVACComponent = hVACComponent.get
 
-          # change and report high speed cop
-          initial_cop = hVACComponent.ratedCOP
+          # change and report high speed cop. Prior to 3.5.0 it was an OptionalDouble
+          initial_cop = OpenStudio::OptionalDouble.new(hVACComponent.ratedCOP)
```

Comment:
COP for CoilCoolingDX(Single|Two)Speed is now a double, not an OptionalDouble. To maintain backward compat, wrap it into an OptionalDouble anyways, it'll work before/after